nixos/modules/system/boot/networkd.nix

```diff
@@ -235,6 +235,21 @@ let
     (assertValueOneOf "AutoJoin" boolValues)
   ];
 
+  checkRoutingPolicyRule = checkUnitConfig "RoutingPolicyRule" [
+    (assertOnlyFields [
+      "TypeOfService" "From" "To" "FirewallMark" "Table" "Priority"
+      "IncomingInterface" "OutgoingInterface" "SourcePort" "DestinationPort"
+      "IPProtocol" "InvertRule" "Family"
+    ])
+    (assertRange "TypeOfService" 0 255)
+    (assertRange "FirewallMark" 1 4294967295)
```

Comment:
Hmm doesn't this break eval with Nix <2.2 on 32bit machines? See also #45722, the issue has been fixed in Nix in https://github.com/NixOS/nix/pull/2378, but AFAICS this is only part of Nix 2.2 (and according to `lib/minver.nix` we support nixpkgs evaluation with at least Nix 2.0).

Reply:
Hah! Did not know about that.

Removing the check then :(